tests/Unit/PointwiseFunctions/AnalyticSolutions/Poisson/Moustache.py

```diff
@@ -7,25 +7,28 @@
 def poly(x, coeffs):
     return np.sum(c * x**p for p, c in enumerate(coeffs))
 
+
 def field(x):
     return np.prod(x * (1. - x)) * np.sum((x - 0.5)**2)**(3. / 2.)
 
-def auxiliary_field(x):
+
+def field_gradient(x):
     if len(x) == 1:
         return np.abs(x - 0.5) * poly(x, [0.25, -3, 7.5, -5])
     elif len(x) == 2:
         return np.sqrt(np.sum((x - 0.5)**2)) * np.array([
             x[d - 1] * (1. - x[d - 1]) * (poly(x[d], [
                 0.5, poly(x[d - 1], [-3.5, 2, -2]), 7.5, -5
-                ]) + poly(x[d - 1], [0, -1, 1]))
+            ]) + poly(x[d - 1], [0, -1, 1]))
```

Comment:
Just a nitpick: I'm unclear why the indentation changed here

Reply:
`autopep8` probably did that. I can revert the change

Reply:
squashed this in to revert the change